src/app/fsat/fsat-tabs/fsat-tabs.component.ts

```diff
@@ -179,7 +179,14 @@ export class FsatTabsComponent implements OnInit {
     let fluidValid: boolean = this.fsatFluidService.isFanFluidValid(this.fsat.baseGasDensity);
     let fanValid: boolean = this.fanSetupService.isFanSetupValid(this.fsat.fanSetup, true);
     let motorValid: boolean = this.fanMotorService.isFanMotorValid(this.fsat.fanMotor);
-    let motorWarnings: FanMotorWarnings = this.fsatWarningService.checkMotorWarnings(this.fsat, this.settings);
+    let isMod: boolean;
```

Comment:
Don't need this. Fsat Tabs is always be for baseline. Modificaitons have their own tabs.